proxy_map.go

```diff
@@ -908,20 +930,29 @@ func (m *Map) loadAll(ctx context.Context, replaceExisting bool, keys ...interfa
 	if len(keys) == 0 {
 		request = codec.EncodeMapLoadAllRequest(m.name, replaceExisting)
 	} else {
-		keyDatas := make([]*serialization.Data, 0, len(keys))
-		for _, key := range keys {
-			if keyData, err := m.convertToData(key); err != nil {
-				return err
-			} else {
-				keyDatas = append(keyDatas, keyData)
-			}
+		keyDatas, err := m.convertToDataList(keys)
+		if err != nil {
+			return err
 		}
 		request = codec.EncodeMapLoadGivenKeysRequest(m.name, keyDatas, replaceExisting)
 	}
 	_, err := m.invokeOnRandomTarget(ctx, request, nil)
 	return err
 }
 
+func (m *Map) convertToDataList(keys []interface{}) ([]*serialization.Data, error) {
+	keyDatas := make([]*serialization.Data, 0, len(keys))
+	for _, key := range keys {
+		// todo check if this null check done with validateAndSerialize cause problem for keys
+		if keyData, err := m.validateAndSerialize(key); err != nil {
+			return nil, err
+		} else {
+			keyDatas = append(keyDatas, keyData)
```

Comment:
Can you change this to:
```
keyData, err := err := m.validateAndSerialize(key)
if err != nil  {
   return nil, err
}
keyDatas = append(keyDatas, keyData)
```

Reply:
I will be fixing that, thanks for that catch. I wasn't sure to use `validateAndSerialize` with alternative being `convertToData` which is essentially same minus a nil check. 
I know that we support "null" as value on our map structure but do we support null keys? If not I validateAndSerialize seems fine here

Reply:
No, we don't support `nil` for neither the key nor the value. `validateAndSerialize` is fine here. I think just reformatting that code is fine.